### ISSUE TITLE
When run with --info, script crashes out when trying to parse version_strings

### ISSUE THREAD

**Issue by eriksejr** [P1sec/QCSuper#35]
Hello,
Thanks again for your work on this. I suspect this is a relatively easy fix, I might even take a whack at in when I have time though python is not really my thing. The USB modem is a sierra wireless MC7455 module. When the script is run with --info on this device it returns the following output:

```
./qcsuper.py --info --usb-modem /dev/ttyUSB0

[+] Compilation date:    Jul 24 2018 18:20:25
[+] Release date:        May 02 2018 05:00:00
[+] Version directory:   C2300300

[+] Common air interface information: 
[+]   Station classmark: 58
[+]   Common air interface revision: 6
[+]   Mobile model:      255
[+]   Mobile firmware revision: 100
[+]   Slot cycle index:  2
[+]   Hardware revision: 0x922 (9.34)

Traceback (most recent call last):
  File "/home/pi/qcsuper/inputs/_base_input.py", line 218, in _init_single_module
    module.on_init()
  File "/home/pi/qcsuper/modules/info.py", line 88, in on_init
    build_id, model_string, _ = ver_strings.split(b'\x00')
ValueError: too many values to unpack (expected 3)

```
This seems to be incomplete, it looks like it's getting something within the version string that it does not expect.

If I can be of more help gathering further info on this to help improve the script please let me know.

Thanks!

**Comment by StefanBruens** [comment]
Same on Sierra MC7710.

@eriksejr - can you change line 88 of info.py as follows:
```
-            build_id, model_string, _ = ver_strings.split(b'\x00')
+            print_row('ver_strings', ver_strings)
+            build_id, model_string, *_ = ver_strings.split(b'\x00')
```
For the MC7710, output is:
`[+] ver_strings:         b'M9200B-SCAQWBZD-X3052903\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'`

As can be seen, the output is padded with `b'\x00'`, each creating an empty segment.

**Comment by eriksejr** [comment]
@StefanBruens yes that fixes the issue. The version string is similar padded with NULLs:

```
[+] Compilation date:    May 19 2017 06:37:30
[+] Release date:        Apr 21 2017 10:00:00
[+] Version directory:   C2240506

[+] Common air interface information: 
[+]   Station classmark: 58
[+]   Common air interface revision: 6
[+]   Mobile model:      255
[+]   Mobile firmware revision: 100
[+]   Slot cycle index:  2
[+]   Hardware revision: 0x922 (9.34)

[+] ver_strings:         b'C2240506-M9635TAAAANAZM-1.96273.1\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
[+] Mobile model ID:     0xffd
[+] Chip version:        2
[+] Firmware build ID:   C2240506-M9635TAAAANAZM-1.96273.1

[+] Diag version:        8

[+] Serial number:       0
```
Thanks!

**Comment by StefanBruens** [comment]
@eriksejr - you shouldn´t close this, as long as a fix is not committed to the repo.

**Comment by p1-mmr** [comment]
This should be fixed in commit 586f3c0, thanks
